src/org/zaproxy/zap/extension/quickstart/ExtensionQuickStart.java

```diff
@@ -432,4 +436,12 @@ public void handleNoSavedReport() {
 			}
 		}
 	}
+
+	@Override
+	public void optionsChanged(OptionsParam optionsParam) {
+		//PnH button has the same enable state as the API
+		getQuickStartPanel().updateConfButton(optionsParam.getApiParam().isEnabled());
+		getQuickStartPanel().updateConfField(optionsParam.getApiParam().isEnabled());

```

Comment:
From outside of QuickStartPanel the button/field should be changed with a single method, to ensure that the button/field enabled state are always in sync.